bitnami/concourse/values.yaml

```diff
@@ -0,0 +1,1202 @@
+## @section Global parameters
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets and storageClass
+
```

Comment:
These values were in the template but in values.yaml (there are false positives):

```
Key concourse.web.baseResourceTypeDefaults not found
Key concourse.worker.baggageclaim.logLevel not found
Key externalDatabase.postgresqlUsername not found
Key podSecurityPolicy.allowedWebVolumes not found
Key postgresql.nameOverride not found
Key rbac.apiVersion not found
Key secrets.workerAdditionalCerts not found
Key web.debugBindPort not found
Key web.enableAcrossStep not found
Key web.enableCacheStreamedVolumes not found
Key web.enablePipelineInstances not found
Key web.ingress.certManager not found
Key web.ingress.extraHosts not found
Key web.ingress.extraTls not found
Key web.ingress.hostname not found
Key web.ingress.path not found
Key web.ingress.pathType not found
Key web.tsa.bindDebugPort not found
Key worker.autoscaling.builtInMetrics not found
Key worker.autoscaling.customMetrics not found
Key worker.autoscaling.maxReplicas not found
Key worker.autoscaling.minReplicas not found
Key worker.command not found
Key worker.customLivenessProbe not found
Key worker.customReadinessProbe not found
Key worker.hostAliases not found
Key worker.nodeSelector not found
Key worker.readinessProbe.enabled not found
Key worker.tolerations not found
```